modules/backend/widgets/Table.php

```diff
@@ -131,6 +142,8 @@ public function prepareVars()
         $this->vars['deleting'] = $this->getConfig('deleting', true);
         $this->vars['toolbar'] = $this->getConfig('toolbar', true);
         $this->vars['height'] = $this->getConfig('height', false) ?: 'false';
+        $this->vars['minItems'] = (int) $this->getConfig('minItems', 1);
+        $this->vars['maxItems'] = (int) $this->getConfig('maxItems');
```

Comment:
This would convert the null to 0 would it not?

Reply:
That should be fine - zero is considered no limit.